override/merge.go

```diff
@@ -180,6 +195,16 @@ func mergeUlimit(_ any, o any, p tree.Path) (any, error) {
 	return o, nil
 }
 
+func mergeIPAMConfig(c any, o any, path tree.Path) (any, error) {
+	right := convertIntoMapping(c.([]any)[0], nil)
```

Comment:
I don't get the logic here. Why only merge the first element ?
IIUC we should merge IPAM configs within the same network segment (same `subnet`) and append the others

Reply:
Rohh my bad 🤦

Reply:
I'll fix that Monday 😞